### ISSUE TITLE
Rock.Bus.BusException

### ISSUE THREAD

**Issue by mikedotmundy** [SparkDevNetwork/Rock#5005]
### Description

We have been seeing this exception several times daily since updating to v13. I originally thought these were because of issue #4920, but it seems they are hitting more often than just for that. 

Each time it happens, 2 separate sets of exceptions are thrown, and it looks like this in my exception log:
<img width="1590" alt="Screen Shot 2022-05-09 at 21 25 49" src="https://user-images.githubusercontent.com/23226974/167524188-c36414f3-777e-48f7-bbbd-4c38650a5aa1.png">

#### First exception:
<img width="1560" alt="Screen Shot 2022-05-09 at 21 26 42" src="https://user-images.githubusercontent.com/23226974/167524261-53c26038-d894-451e-8cf5-2efd30dd00e3.png">

First Stack Trace:
```
at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Rock.Tasks.ProcessEntityTypeIndex.Execute(Message message) in C:\Github\Rock13\Rock\Tasks\ProcessEntityTypeIndex.cs:line 56
   at Rock.Bus.Consumer.RockConsumer`2.Consume(ConsumeContext`1 context) in C:\Github\Rock13\Rock\Bus\Consumer\RockConsumer.cs:line 87
   at MassTransit.Pipeline.ConsumerFactories.DelegateConsumerFactory`1.d__2`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MassTransit.Pipeline.Filters.ConsumerMessageFilter`2.>-Send>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at MassTransit.Pipeline.Filters.ConsumerMessageFilter`2.>-Send>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GreenPipes.Filters.TeeFilter`1.d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GreenPipes.Filters.OutputPipeFilter`2.d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at GreenPipes.Filters.OutputPipeFilter`2.d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GreenPipes.Filters.DynamicFilter`1.d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MassTransit.Pipeline.Filters.DeserializeFilter.d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GreenPipes.Filters.RescueFilter`2.-Send>d__5.MoveNext()
```
Second Stack Trace:
```
at Rock.Model.Group.IndexDocument(Int32 id) in C:\Github\Rock13\Rock\Model\Group\Group\Group.Logic.cs:line 145
```

#### Second exception:
<img width="1561" alt="Screen Shot 2022-05-09 at 21 22 37" src="https://user-images.githubusercontent.com/23226974/167523866-e2e37942-3871-4795-b90a-03b54bd2dd4a.png">

Both stack traces are identical to above.

These 2 exceptions seem to have to do with Groups.
Here is another one that looks very similar to the above, but is something to do with Content Channels.

#### First Exception:
<img width="1556" alt="Screen Shot 2022-05-09 at 21 28 47" src="https://user-images.githubusercontent.com/23226974/167524448-48604f98-4479-4636-b8c1-8448bb38616d.png">

First Stack Trace is identical to the one above
Second Stack Trace:
```
at Rock.Model.ContentChannelItem.IndexDocument(Int32 id) in C:\Github\Rock13\Rock\Model\CMS\ContentChannelItem\ContentChannelItem.cs:line 340
```

#### Second Exception
<img width="1553" alt="Screen Shot 2022-05-09 at 21 31 10" src="https://user-images.githubusercontent.com/23226974/167524668-8c581125-0191-4494-a981-447245a448c4.png">

First Stack Trace identical to above.
Second stack trace:
```
at Rock.Model.ContentChannelItem.IndexDocument(Int32 id) in C:\Github\Rock13\Rock\Model\CMS\ContentChannelItem\ContentChannelItem.cs:line 340
```

### Versions

* **Rock Version:** [v13.3]
* **Client Culture Setting:** [en-US]

**Comment by asdf1nit** [comment]
I have all of the above happening on our system as well. Exactly as Mike laid it out.

**Comment by leahjennings** [comment]
We also have these exceptions in our system.

**Comment by nairdo** [comment]
We were able to reproduce this one and feel pretty confident in a solution which we're working on for the imminent v13.4 alpha release.

**Comment by ethan-sparkdevnetwork** [comment]
This is fixed in commit 1dddbb52.
